lib/spack/spack/cmd/bootstrap.py

```diff
@@ -22,84 +22,71 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
+import argparse
 import os
 
+import llnl.util.filesystem as fs
 import llnl.util.tty as tty
-from llnl.util.filesystem import join_path, mkdirp
-
 import spack
-from spack.util.executable import ProcessError, which
-
-_SPACK_UPSTREAM = 'https://github.com/llnl/spack'
+import spack.cmd
+import spack.cmd.common.arguments as arguments
+from spack.build_environment import InstallError
+from spack.fetch_strategy import FetchError
+from spack.package import PackageBase
 
-description = "create a new installation of spack in another prefix"
+from spack.util.executable import which
 
+description = "bootstrap packages needed for spack to run smoothly."
```

Comment:
Descriptions should not end in a period to match what `argparse` prints by default.